### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by elhamatokhi**
-  In this branch, I have added the contact section to the HTML file.
- I have added some CSS styles to the contact section.

### CODE REVIEW

**Review by Mre55 — COMMENTED** [review]
**Hi @elhamatokhi,**

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :negative_squared_cross_mark:

You have submitted a project, that has no `README.md` file. You can this [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) and edit to your specific project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

**Review by bertinkambale — COMMENTED** [review]
Hi @elhamatokhi 
Good job so far! I know you have worked hard on making this milestone There are some issues that you still need to work on to go to the next project but you are almost there!
-Kindly do not forget to  take into consideration the commit I mentioned above in your Html and CSS

File: index.html [hunk 1]
```diff
@@ -152,140 +152,268 @@ <h2 class="title">Tonic</h2>
...
+              required
+            />
           </div>
-        </section>
-      <!--About my self section end-->
-
-
-
-
-
```
Comment (zieeco) [inline index.html]: ***[OPTIONAL]***

- Please, let's keep our codebase clean by removing the unnecessary added spaces to control the use of indentation and to make our codebase more readable

File: style.css [hunk 2]
```diff
@@ -346,3 +345,89 @@ summary:hover .summary-up svg {
...
+  width: 100%;
+  height: 779px;
+  background-color: #6070ff;
+  background-image: url("Images/contact\ form\ shapes-mobile@2x.png");
+  background-repeat: no-repeat;
+  background-size: auto;
+  background-position: 0 50px;
+  border-radius: 90px 0 0 0;
+  z-index: -999;
+}
```
Comment (zieeco) [inline style.css]: **[REQUIRED CHANGES]**

- Please, let's use the right background image for our contact form to match the figma design
- Please, let's also add the missing black line to match the figma design

File: README.md [hunk 3]
```diff
@@ -0,0 +1 @@
+# portfolio-micro
```
Comment (zieeco) [inline README.md]: **[REQUIRED CHANGES]**

- Please, let's update our readme to make it more professional, as of right now it missing some important headers. refer to the[microverse readme template](https://github.com/microverseinc/readme-template) to add the missing header to make it more professional

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Team


Great work on the contact form 👏 
You've done well implementing it, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
_Check the comments under the review._

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (elhamatokhi) [reply index.html]: Thanks for the comment, but I do not think I had this in my code. if there was, so why I did not receive any linter error?

File: README.md (hunk 3 above)
Comment (elhamatokhi) [reply README.md]: Thanks, I just delete it back. Because I do not think we need a README file in a branch.

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @elhamatokhi  😃,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
-  The form is submitted successfully with valid input values. Great job 👍

 ## Required Changes ♻️
- [ ] Your repo doesn't have a Readme file.  It will be nice if you could add a descriptive README file to your project. That file will give visitors a brief understanding of your project. Please use the [Microver template](https://github.com/microverseinc/readme-template) as an example.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @elhamatokhi 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**